system/ee/ExpressionEngine/Addons/rte/views/redactor-toolbar.php

```diff
@@ -0,0 +1,28 @@
+<div id="rte-toolbar">
+    <div class="form-standard">
+        <div class="redactor-toolbar">
+            <h2>Toolbar Buttons</h2>
+            <?php foreach ($buttons as $btnHandle => $btn) :?>
+                <p>
+                    <label><?=$btn['label']?></label>
+                    <?=form_checkbox('settings[toolbar][buttons][]', $btnHandle, in_array($btnHandle, $settings['toolbar']['buttons']))?>
+                </p>
+            <?php endforeach;?>
+        </div>
+        
+        <div class="redactor-toolbar">
+            <h2>Plugins</h2>
```

Comment:
The H2 headings (just as any text) needs to come from lang files

Reply:
Thank you, fixed in latest commit.